packages/loopring_v3/contracts/aux/access/LoopringIOExchangeOwner.sol

```diff
@@ -123,13 +131,27 @@ contract LoopringIOExchangeOwner is SelectorBasedAccessManager, ERC1271, Drainab
         // Decode the blocks
         ExchangeData.Block[] memory blocks = _decodeBlocks(decompressed);
 
-        // Process the callback logic.
-        _beforeBlockSubmission(blocks, config);
```

Comment:
It may be useful, but I don't know if we should add support for things that we currently don't plan to use (and if we use it maybe it needs additional changes again).

But a boolean before/after may indeed be useful with perhaps very little work, so perhaps still worth it.

Reply:
The normal callbacks can now be run before or after the blocks submission using a flag.

I kept the `TransactionReceiver`s (AMM, Bridge) to after the block submissions only to keep things simple (a bit more difficult to do then for the normal callbacks).